.github/workflows/release.yml

```diff
@@ -0,0 +1,161 @@
+name: Release Flow
+
+.publish_base: &publish
+  uses: pypa/gh-action-pypi-publish@release/v1
+  with: &pypi-config
+    user: __token__
+    password: ${{ secrets.PYPI_API_TOKEN }}
+    skip_existing: true
+
+# Controls when the action will run.
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish-pip-packages:
+    name: Create Release
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.10"]
+    steps:
+      - name: Get version from tag
+        id: tag_name
+        run: |
+          echo ::set-output name=current_version::${GITHUB_REF#refs/tags/v}
+        shell: bash
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-versions }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pipenv twine
+
+      - name: New Environment
+        run: |
+          make new_env
+
+      - name: Build wheels and source tarball
+        run: >-
+          echo Processing open-aea
+          make dist
+
+          echo Processing aea-cli-benchmark
+          cd plugins/aea-cli-benchmark
+          python setup.py sdist bdist_wheel
+          cd ../..
+
+          echo Processing aea-cli-ipfs
+          cd plugins/aea-cli-ipfs
+          python setup.py sdist bdist_wheel
+          cd ../..
+
+          echo Processing aea-ledger-cosmos
+          cd plugins/aea-ledger-cosmos
+          python setup.py sdist bdist_wheel
+          cd ../..
+
+          echo Processing aea-ledger-ethereum
+          cd plugins/aea-ledger-ethereum
+          python setup.py sdist bdist_wheel
+          cd ../..
+
+          echo Processing aea-ledger-fetchai
+          cd plugins/aea-ledger-fetchai
+          python setup.py sdist bdist_wheel
+          cd ../..
+
+
+      - name: Publish Package to PyPI
+        <<: *publish
+
+      - name: Publish cli-benchmark Plugin to PyPI
+        <<: *publish
+          <<: *pypi-config
+            packages_dir: plugins/aea-cli-benchmark/dist
```

Comment:
Ref: https://github.com/trzsz/trzsz/blob/cf36cdf29330ed930c37336abddaf3bb931c532d/.github/workflows/publish-to-pypi.yml#L45

Reply:
Aww you didnt reference my issue :'(